wrk3.go

```diff
@@ -150,7 +166,7 @@ func (e *executioner) sendRequests() {
 		case t, ok := <-e.eventsBuf:
 			if ok {
 				res.counter++
-				err := e.benchmark.SendRequest()
+				err := e.benchmark.SendRequest.ExecuteRequest(res.counter)
```

Comment:
Basically this means all go routines cycle through the same clients at roughly the same times (and if the index is used for requests, for that as well.

Reply:
roughly is not the same and in any way so what ? the whole point here is to allow concurrent requests to the server.
the index is just an indication of progress, not even a global one and the way that I use it in the test allow me to distribute over the connections. in my book it is good enough.